tensorboard/webapp/runs/views/runs_table/runs_data_table.scss

```diff
@@ -12,8 +12,16 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
+@use '@angular/material' as mat;
+@import 'tensorboard/webapp/theme/tb_theme';
+
 $_circle-size: 12px;
 
+:host {
+  overflow-y: scroll;
+  width: 100%;
+}
```

Comment:
This is fixing the styling on the table component itself, it was way too small before.